esnext-browsers.js

```diff
@@ -284,6 +284,18 @@ module.exports = {
       short: 'CH 52',
       note_id: 'experimental-flag',
   },
+  chrome53: {
```

Comment:
Is Chrome 53 really needed here?  There's no other mention to this version in the test results, I don't see the point of the extra column.